packages/gqless/src/Client/client.ts

```diff
@@ -143,14 +143,14 @@ export interface Mutate<
     opts?: {
       onComplete?: (data: T, helpers: MutateHelpers<GeneratedSchema>) => void;
       onError?: (
-        error: gqlessError,
+        error: GQlessError,
         helpers: MutateHelpers<GeneratedSchema>
       ) => void;
     }
   ): Promise<T>;
 }
 
-export interface GqlessClient<
+export interface GQlessClient<
```

Comment:
This type name change is a breaking change and the previous version should be also provided with the `@deprecated` comment